### DISCUSSION TITLE
I would like to display a limited list of countries

### DISCUSSION THREAD

**Discussion by victorgoecking** [AstrOOnauta/react-native-international-phone-number#20] [Q&A] [answered]
My app will work with only 4 countries, Ex.: ['BR', 'PT', 'CA', 'US']. 
Is there any way to display only these in the list of countries for selection?

**Comment by AstrOOnauta** [comment, marked as answer]
Hey @victorgoecking , sorry for my delay...

About your question, great!

The v0.6.0 not support this, but I released a next version that support a feat `showOnly` and `excludedCountries`.

Please, update the lib version to v0.6.1 in your project, read the next point in the [docs](https://github.com/AstrOOnauta/react-native-international-phone-number#show-only-some-countries-inside-modal) to resolve your issue and enjoy!!

🚀AstrOOnauta 🚀

**Reply by victorgoecking** [reply]
Thank you very much
